yarn.lock

```diff
@@ -2481,6 +2500,13 @@ available-typed-arrays@^1.0.5:
   resolved "https://registry.yarnpkg.com/available-typed-arrays/-/available-typed-arrays-1.0.5.tgz#92f95616501069d07d10edb2fc37d3e1c65123b7"
   integrity sha512-DMD0KiN46eipeziST1LPP/STfDU0sufISXmjSgvVsoU2tqxctQeASejWcfNtxYKqETM1UxQ8sp2OrSBWpHY6sw==
 
+axios@^0.21.1:
```

Comment:
Can we get rid of that one?

Reply:
@louisinger axios is listed as both a dependency and a dev dependency in liquidjs-lib. It seems it is not used except in the tests, could you convert it to just a dev dependency and let us know when it's updated?

Reply:
Oh right ! nice catch thanks, here is the [PR](https://github.com/vulpemventures/liquidjs-lib/pull/105), should be released soon

Reply:
@rolznz liquidjs-lib `6.0.2-liquid.29` has been released without axios

Reply:
@louisinger thanks!